Frontend/ui-common/src/Engineering/Helpers/FileProvider/Utils.js

```diff
@@ -0,0 +1,65 @@
+export const loadCommonConfig = function () {
+  return JSON.stringify(defaultConfig);
+}
```

Comment:
are we using this function ?

Reply:
and from where the `defaultConfig ` is coming

Reply:
It is not used. Removing